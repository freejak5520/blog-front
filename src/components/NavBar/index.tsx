import Link from "next/link";

const NavBar = () => {
  return (
    <div className="flex flex-row justify-end gap-3">
      <Link className="font-semibold" href={"/posts"}>
        Blog
      </Link>
      <Link className="font-semibold" href={"/about"}>
        About
      </Link>
      <Link
        className="font-semibold"
        href={"https://github.com/freejak5520"}
        target="_blank"
      >
        GitHub
      </Link>
    </div>
  );
};

export default NavBar;
