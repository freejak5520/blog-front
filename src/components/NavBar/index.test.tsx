import { render } from "@testing-library/react";
import NavBar from ".";

describe("NavBar", () => {
  it("render test", () => {
    const { container } = render(<NavBar></NavBar>);

    expect(container).toMatchSnapshot();
  });
});
