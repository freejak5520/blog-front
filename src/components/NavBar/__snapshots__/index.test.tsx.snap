// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar render test 1`] = `
<div>
  <div
    class="flex flex-row justify-end gap-3"
  >
    <a
      class="font-semibold"
      href="/posts"
    >
      Blog
    </a>
    <a
      class="font-semibold"
      href="/about"
    >
      About
    </a>
    <a
      class="font-semibold"
      href="https://github.com/freejak5520"
      target="_blank"
    >
      GitHub
    </a>
  </div>
</div>
`;
